monai/networks/blocks/upsample.py

```diff
@@ -71,3 +71,64 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             x: Tensor in shape (batch, channel, spatial_1[, spatial_2, ...).
         """
         return torch.as_tensor(self.upsample(x))
+
+
+class SubpixelUpsample(nn.Module):
+    """
+    Upsample via using a subpixel CNN. This module supports 1D, 2D and 3D input images.
+    The module is consisted with two parts. First of all, a convolutional layer is employed
+    to increase the number of channels into: ``in_channels * (scale_factor ** spatial_dims)``.
+    Secondly, a pixel shuffle manipulation is utilized to aggregates the feature maps from
+    low resolution space and build the super resolution space.
+    The first part of the module is not fixed, a sequential layers can be used to replace a
+    single layer. 
+    The idea comes from:
```

Comment:
not fixing the conv part will provide more flexibility, but seems it's not implemented yet in this PR?

Reply:
yes, not implement. I'm not sure how to write down a flexible structure here, cuz users can use 1 or 2 or many different layers before the pixel shuffle part.